src/ray/core_worker/core_worker.h

```diff
@@ -278,6 +278,11 @@ class CoreWorker {
   /// \return void.
   void StartExecutingTasks();
 
+  Status GetReturnObjects(const std::vector<ObjectID> object_ids,
```

Comment:
it should be `const std::vector<ObjectID>& object_ids` and `const std::vector<size_t> &data_sizes` here.

Reply:
Interesting, I have references in cython and not here but it compiles & works...